linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -101,32 +95,12 @@ private void run(String groupId, String artifactId)
 
     Set<SymbolProblem> fixedProblems = Sets.difference(problemsInBaseline, problemsInSnapshot);
     if (!fixedProblems.isEmpty()) {
-      int problemSize = fixedProblems.size();
-      StringBuilder message =
-          new StringBuilder(
-              "The following problem"
-                  + (problemSize > 1 ? "s" : "")
-                  + " in the baseline no longer appear in the snapshot:\n");
-      for (SymbolProblem problem : fixedProblems) {
-        message.append(problem + "\n");
-      }
-      System.out.println(message.toString());
+      System.out.println(formatMessageForFixedError(fixedProblems));
```

Comment:
I suspect System.out.println is not the right place to putput, but we can think about that later. Maybe a logger? I'm not sure.